src/all/deviantart/src/eu/kanade/tachiyomi/extension/all/deviantart/DeviantArt.kt

```diff
@@ -0,0 +1,163 @@
+    override val baseUrl = "https://deviantart.com"
+    override val lang = "all"
+    override val supportsLatest = false
+
+    private val backendBaseUrl = "https://backend.deviantart.com"
+    private fun backendBuilder() = backendBaseUrl.toHttpUrl().newBuilder()
+
+    private val dateFormat by lazy {
+        SimpleDateFormat("EEE, dd MMM yyyy HH:mm:ss zzz", Locale.ENGLISH)
+    }
+
+    private fun parseDate(dateStr: String): Long {
+        return try {
+            dateFormat.parse(dateStr)!!.time
+        } catch (_: ParseException) {
+            0L
+        }
+    }
+
+    override fun popularMangaRequest(page: Int): Request {
+        throw UnsupportedOperationException(SEARCH_ERROR_MSG)
+    }
+
+    override fun popularMangaParse(response: Response): MangasPage {
+        throw UnsupportedOperationException(SEARCH_ERROR_MSG)
+    }
+
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
+        require(query.startsWith("gallery:")) { SEARCH_ERROR_MSG }
+        try {
+            val (username, folderId) = query.substringAfter(":").split("/")
+            return GET("$baseUrl/$username/gallery/$folderId", headers)
+        } catch (_: IndexOutOfBoundsException) {
+            throw IllegalArgumentException(SEARCH_ERROR_MSG)
+        }
+    }
+
+    override fun searchMangaParse(response: Response): MangasPage {
+        val manga = mangaDetailsParse(response)
+        return MangasPage(listOf(manga), false)
+    }
+
+    override fun latestUpdatesRequest(page: Int): Request {
+        throw UnsupportedOperationException()
+    }
+
+    override fun latestUpdatesParse(response: Response): MangasPage {
+        throw UnsupportedOperationException()
+    }
+
+    override fun mangaDetailsParse(response: Response): SManga {
+        val document = response.asJsoup()
+        val subFolderGallery = document.selectFirst("#sub-folder-gallery")
+        val manga = SManga.create().apply {
+            // If manga is sub-gallery then use sub-gallery name, else use gallery name
+            title = subFolderGallery?.selectFirst("._2vMZg + ._2vMZg")?.text()?.substringBeforeLast(" ")
+                ?: document.selectFirst(".ds-card-selected h2")!!.text()
+            author = document.title().substringBefore(" ")
+            description = subFolderGallery?.selectFirst(".legacy-journal")?.wholeText()
+            thumbnail_url = subFolderGallery?.selectFirst("img[property=contentUrl]")?.absUrl("src")
+        }
+        manga.setUrlWithoutDomain(response.request.url.toString())
+        return manga
+    }
+
+    override fun chapterListRequest(manga: SManga): Request {
+        val pathSegments = getMangaUrl(manga).toHttpUrl().pathSegments
+        val username = pathSegments[0]
+        val folderId = pathSegments[2]
+
+        val url = backendBuilder()
+            .addPathSegment("rss.xml")
+            .addQueryParameter("q", "gallery:$username/$folderId")
+            .build()
```

Comment:
Fails for `gallery:millaswann/all`, should either not allow the entry to be shown in the first place (`searchMangaParse`) or somehow account for it.